README.rst

```diff
@@ -32,6 +32,7 @@ Requirements:
 - pyglet (a bleeding-edge ``tip.zip`` as of the end of May 2014), see the
   "development version" here:
     http://www.pyglet.org/download.html
+- TDTPy (only needed if using Windows)
```

Comment:
Related reading on old repo page, in case you're bored and curious:

https://bitbucket.org/bburan/tdtpy/pull-request/1/fix-encodings-in-files/diff
https://bitbucket.org/bburan/tdtpy/issue/3/some-files-use-mbcs-encoding
https://bitbucket.org/bburan/tdtpy/issue/7/attributeerror-in-circuit-info-retrieval
https://bitbucket.org/bburan/tdtpy/issue/6/no-device-id-parameter-for-circuits-in